docs/FHIR-anonymization.md

```diff
@@ -441,96 +473,128 @@ To generalize dateTime, time, date and instant type using expression
+* [optional] **rangeType** Define whether the *span* value is *fixed* or *proportional*. The default value is *fixed*. 
+* [optional] **roundTo** A value from 0 to 28 that specifies the number of decimal places to round to. The default value is *0* for integer types and *2* for decimal types. 
+
+> **[NOTE]**:
 > The target field should be of either a numeric type (integer, decimal, unsignedInt, positiveInt) or a quantity type (Quantity, SimpleQuantity, Money, etc.). 
 
 ### Generalize
+
 As one of the anonymization methods, generalization means mapping values to the higher level of generalization. It is the process of abstracting distinguishing value into a more general, less distinguishing value. Generalization attempts to preserve data utility while also reducing the identifiability of the data. 
 Generalization uses FHIRPath predicate expression to define a set of cases that specify the condition and target value like [sample rules](#Sample-rules-using-FHIRPath). Follows are some examples of cases.
 
 |Data Type|Cases|Explanation|Input data-> Output data|
 |-----|-----|-----|-----|
-|numeric|_"$this>=0 and $this<20": "20"_|Data fall in the range [0,20) will be replaced with 20. |18 -> 20|
-|numeric|_"true": "($this div 10)*10"_|Approximate data to multiples of 10. |18 -> 10|
-|string| _"$this in ('es-AR' \| 'es-ES' \| 'es-UY')": "'es'"_|Data fall in the value set will be mapped to "es".|'es-UY' -> 'es'|
-|string| _"$this.startsWith(\'123\')": "$this.subString(0,2)+\'*\*\*\*\' "_ |Mask sensitive string code.|'1230005' -> '123****'|
-|date, dateTime, time|_"$this >= @2010-1-1": "@2010"_|Data fall in a date/time/dateTime range will be mapped to one date/time/dateTime value.| 2016-03-10 -> 2010|
-|date, dateTime, time|"$this.replaceMatches('(?&lt;year&gt;\\\d{2,4})-(?&lt;month&gt;\\\d{1,2})-(?&lt;day&gt;\\\d{1,2})\\\b', '${year}-${month}'"|Omit "day" to generalize specific date.|2016-01-01 -> 2016-01|
+|numeric|`"$this>=0 and $this<20": "20"`|Data fall in the range [0,20) will be replaced with 20. |18 -> 20|
+|numeric|`"true": "($this div 10)*10"`|Approximate data to multiples of 10. |18 -> 10|
+|string| `"$this in ('es-AR' \| 'es-ES' \| 'es-UY')": "'es'"`|Data fall in the value set will be mapped to "es".|'es-UY' -> 'es'|
+|string| `"$this.startsWith(\'123\')": "$this.subString(0,2)+\'*\*\*\*\' "` |Mask sensitive string code.|'1230005' -> '123****'|
+|date, dateTime, time|`"$this >= @2010-1-1": "@2010"`|Data fall in a date/time/dateTime range will be mapped to one date/time/dateTime value.| 2016-03-10 -> 2010|
+|date, dateTime, time|`"$this": "$this.toString().replaceMatches('(?<year>[0-9]{2,4})-(?<month>[0-9]{1,2})-(?<day>[0-9]{1,2})', '${year}-${month})'"`|Omit "day" to generalize specific date.|2016-01-01 -> 2016-01|
```

Comment:
It works if not using toString() here.

Reply:
That raises an exception for me saying it can't be datetime can't be cast to .NET string type.

Reply:
@sowu880 I tested it again and I still have an exception. That is maybe due to a not perfectly formed date, no idea. Still, with the toString, it does always work fine. Shall we merge this PR?